libs/ledger-live-common/src/families/algorand/logic.ts

```diff
@@ -1,48 +1,2 @@
-import { BigNumber } from "bignumber.js";
-import type { AlgorandOperationMode } from "./types";
-import { getAccount } from "./api";
-
-export const ALGORAND_MAX_MEMO_SIZE = 32;
-export const ALGORAND_MIN_ACCOUNT_BALANCE = 100000;
-
-export const recipientHasAsset = async (
-  recipientAddress: string,
-  assetId: string
-): Promise<boolean> => {
-  const recipientAccount = await getAccount(recipientAddress);
-  return recipientAccount.assets.map((a) => a.assetId).includes(assetId);
-};
-
-export const isAmountValid = async (
-  recipientAddress: string,
-  amount: BigNumber
-): Promise<boolean> => {
-  const recipientAccount = await getAccount(recipientAddress);
-  return recipientAccount.balance.isZero()
-    ? amount.gte(ALGORAND_MIN_ACCOUNT_BALANCE)
-    : true;
-};
-
-export const computeAlgoMaxSpendable = ({
-  accountBalance,
-  nbAccountAssets,
-  mode,
-}: {
-  accountBalance: BigNumber;
-  nbAccountAssets: number;
-  mode: AlgorandOperationMode;
-}): BigNumber => {
-  const minBalance = computeMinimumAlgoBalance(mode, nbAccountAssets);
-  const maxSpendable = accountBalance.minus(minBalance);
-  return maxSpendable.gte(0) ? maxSpendable : new BigNumber(0);
-};
-
-const computeMinimumAlgoBalance = (
-  mode: AlgorandOperationMode,
-  nbAccountAssets: number
-): BigNumber => {
-  const base = 100000; // 0.1 algo = 100000 malgo
-  const currentAssets = nbAccountAssets;
-  const newAsset = mode === "optIn" ? 1 : 0;
-  return new BigNumber(base * (1 + currentAssets + newAsset));
-};
+// Encapsulate for LLD et LLM
+export * from "@ledgerhq/coin-algorand/logic";
```

Comment:
Why not update imports in LLD & LLM ? This is basically postponing some cleanup work for later.

(same comment in other files)

Reply:
Indeed, this is deliberate for now, for multiple reasons including avoiding to much of a "big bang" effect (changes are contained in the newly created package and LLC, not updating LLD, LLM or any other app / project using LLC), staying consistent with what have been done on polkadot so far (same reason of "self contained" changed as explained above).
The next step is, in a second iteration (once this PR is merged in), to cleanup and harmonize both polkadot and algorand packages integrations

Reply:
Indeed, my plan / idea (should be discussed with @sprohaszka-ledger as well) is:
1. merge algorand package PR -> https://github.com/LedgerHQ/ledger-live/pull/3306
2. merge evm package PR -> https://github.com/LedgerHQ/ledger-live/pull/3446

Then, 2 options, the order of which would need to be discussed:
- merge [transport and hw-app dependency rework](https://github.com/LedgerHQ/ledger-live/pull/3219) and adapt algorand and evm packages to it
- cleanup, including items present in PR description under "Todo for a second iteration"